### PR TITLE
Test Repair

### PR DESCRIPTION
**Pull request by AndrewDorn**
Repaired some issues regarding PREFIX.

## Summary of Changes

Test cleanup to resolve issues regarding new PREFIX environment variable.

## Changes

* Fixed issues regarding missing prefix values.
* Minor code cleanup.

## How Has This Been Tested?

Unit tests and some large tests pass locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Unit tests
    - [ ] Integration tests
    - [ ] Manual tests (outlined above)
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
    - [ ] Manual testing/integration testing passes (if forked)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: tasks/pg_utils/test/test_database.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def test_single_query_secretsmanager(self):
             'db_pw': 'secret'
         }
         sql_stmt = 'Select * from mytable'
-        exp_err = ('Database Error. could not translate host name "my.db.host.gov" to address: nodename nor servname provided, or not known\n')
+        exp_err = ('Database Error. could not translate host name "my.db.host.gov" to address: Unknown host\n')
         try:
```
Comment (AndrewDorn) [inline tasks/pg_utils/test/test_database.py:152]: I don't really know why this one changed on us.

File: tasks/db_deploy/test/test_db_deploy.py [hunk 2]
```diff
@@ -208,11 +217,13 @@ def test_regression_test(self,
         script_filename_1 = uuid.uuid4().__str__()
         script_filename_2 = uuid.uuid4().__str__()
         drop_database = 'True'
+        prefix = uuid.uuid4().__str__()
+        os.environ['PREFIX'] = prefix
```
Comment (ifestus) [inline tasks/db_deploy/test/test_db_deploy.py:221]: Do we need to do this with the `setUp` you added?

File: tasks/db_deploy/test/test_db_deploy.py [hunk 3]
```diff
@@ -310,11 +321,13 @@ def test_task_pulls_correct_environment_variables(self,
         db_user = uuid.uuid4().__str__()
         db_port = randint(0, 99999).__str__()
         drop_database = 'True'
+        prefix = uuid.uuid4().__str__()
+        os.environ['PREFIX'] = prefix
```
Comment (ifestus) [inline tasks/db_deploy/test/test_db_deploy.py:325]: Do we need to do this with the setUp you added?

**Review by ifestus — APPROVED** [review]
One or two questions, otherwise LGTM :+1:

File: tasks/db_deploy/test/test_db_deploy.py (hunk 2 above)
Comment (AndrewDorn) [reply tasks/db_deploy/test/test_db_deploy.py:221]: I prefer to have the tests manage their own test values. Theoretically we could just pull setUp's randomized value from the environment, but that would make the test reliant on setUp's implementation.

File: tasks/db_deploy/test/test_db_deploy.py (hunk 3 above)
Comment (AndrewDorn) [reply tasks/db_deploy/test/test_db_deploy.py:325]: I prefer to have the tests manage their own test values. Theoretically we could just pull setUp's randomized value from the environment, but that would make the test reliant on setUp's implementation.

File: tasks/db_deploy/test/test_db_deploy.py (hunk 2 above)
Comment (AndrewDorn) [reply tasks/db_deploy/test/test_db_deploy.py:215]: Nevermind. Removed setUp.

File: tasks/db_deploy/test/test_db_deploy.py (hunk 3 above)
Comment (AndrewDorn) [reply tasks/db_deploy/test/test_db_deploy.py:319]: Nevermind. Removed setUp.

**Pull request merged**